### PR TITLE
1D and 2D prop indices of control vectors and points

### PR DESCRIPTION
**Pull request by dqnykamp**

### CODE REVIEW

**Comment by dqnykamp** [comment]
PR is last 2 commits

**Review by jaltekruse — APPROVED** [review]
+1

File: src/Core/utils/label.js [hunk 1]
```diff
@@ -1,3 +1,109 @@
+export function returnWrapNonLabelsSugarFunction({
```
Comment (jaltekruse) [inline src/Core/utils/label.js:1]: We can think about what we might want to unit test for a function like this as we merge in the jest runner. Would this be too small?

Comment (dqnykamp) [reply src/Core/utils/label.js:1]: Good question.  Seems small.

**Pull request merged**